### DISCUSSION TITLE
Will this only work in Edge?  I'm in the UK and Bing Chat forces me to use Edge.

### DISCUSSION THREAD

**Discussion by manglewurzle** [benf2004/Bing-Chat-History#3] [Q&A] [answered]
Will this only work in Edge?  I'm in the UK and Bing Chat forces me to use Edge.  I prefer Brave.

**Comment by benf2004** [comment, marked as answer]
It should work in brave if you pair it with this extension:

https://chrome.google.com/webstore/detail/bing-chat-for-all-browser/jofbglonpbndadajbafmmaklbfbkggpo
